nucleus/hk2/hk2-config/src/main/java/org/jvnet/hk2/config/ConfigModel.java

```diff
@@ -982,7 +982,7 @@ private static String getMetadataFieldKeyedAs(List<String> strings, String name)
         return ( dv );        
     }
     private static List<String> getMetadataFromDescription(Map<String, List<String>> map, String key) {
-        return map.get(key) == null ? Collections.EMPTY_LIST : map.get(key);
+        return map.get(key) == null ? Collections.emptyList() : map.get(key);
```

Comment:
This ternary expression calls get twice. Could be refactored to a temp variable or the use of getOrDefault method.

Reply:
I changed it to use `getOrDefault()` because I couldn't think of a descriptive name for the temp variable.

Reply:
Thank you.